src/main/scala/performanceanalysis/GetAlertsActor.scala

```diff
@@ -3,30 +3,48 @@ package performanceanalysis
 import akka.actor.{ActorLogging, Actor, Props, ActorRef}
 import performanceanalysis.server.Protocol.{RequestAlertRuleDetails, AllAlertRuleDetails, SingleAlertRuleDetails}
 import performanceanalysis.server.Protocol.Rules.AlertRule
+import scala.concurrent.duration._
 
 /**
   * Created by steven on 17-5-16.
   */
 object GetAlertsActor {
   def props(ruleList: List[ActorRef], originalSender: ActorRef): Props = Props(new GetAlertsActor(ruleList, originalSender))
+
+  val timeOut = 4.seconds
 }
 
 class GetAlertsActor(alertActors: List[ActorRef], originalSender: ActorRef) extends Actor with ActorLogging {
 
+  import context.dispatcher
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://www.codacy.com/app/scala-academy/performance-analysis/file/2667418563/issues/source?bid=3220915&fileBranchId=3331880#l19)